package.json

```diff
@@ -79,5 +78,6 @@
 			"unicorn/prevent-abbreviations": "off"
 		}
 	},
-	"prettier": "@vdemedes/prettier-config"
+	"prettier": "@vdemedes/prettier-config",
+	"packageManager": "pnpm@10.8.0+sha512.0e82714d1b5b43c74610193cb20734897c1d00de89d0e18420aebc5977fa13d780a9cb05734624e81ebd81cc876cd464794850641c48b9544326b5622ca29971"
```

Comment:
Don't make unrelated changes

Reply:
Fixed. Sorry, I didn't notice that it was automatically added to the package.json.